src/tests/functional/shared_test_classes/src/base/ov_subgraph.cpp

```diff
@@ -173,9 +173,16 @@ void SubgraphBaseTest::configure_model() {
     // configure output precision
     {
         auto results = function->get_results();
+        size_t gapSize = results.size() - outType.size();
```

Comment:
in case what precision will be same, the better option to propagate it in separated function (for example, `init_element_types`) and call this function from `configure_model`

Reply:
The precision and layout alignment function is moved into `AlignParameters()` in commit https://github.com/openvinotoolkit/openvino/pull/9975/commits/6c81c83c8978db4597dc0a9505e31a0f4388651e .